Chiazokam/Politico
# #163504409 Create the create party endpoint

#### What does this PR do?
This PR tests and creates the Create party endpoint

#### Description of Task to be completed?
- write tests for every test case
- verify user with jwt
- write validators for the admin input
- create the create party endpoint

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-database-admin-create-party-163504409`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Send a `POST` request to `http://localhost:3000/api/v1/parties `
- You should receive a successful response

#### Any background context you want to provide?
- the POST endpoint payload should be structured like this- 
`
{
	"partyName": "People Congress",
      "partyAddress": "12 Lincoln Ride",
      "partyLogo":"www.logo.com/url.jpg"
}
`
#### What are the relevant pivotal tracker stories?
[#163504409](https://www.pivotaltracker.com/story/show/163504409)

### Images
![image](https://user-images.githubusercontent.com/26940294/51847922-f223ce00-231c-11e9-94ae-737fe55d8565.png)
